extras/sponge.rb

```diff
@@ -96,7 +96,8 @@ def cookies(host)
     end
   end
 
-  def fetch(url, method = :get, fields = nil, raw_post_data = nil, headers = {}, limit = 10)
+  def fetch(url, method = :get, fields = nil, raw_post_data = nil, headers = {}, limit = 10,
```

Comment:
This method already has too many arguments, let's not add one more. Let's return `res` and change existing call sites to `fetch(...).body`.

Reply:
This is the first time I ever write some Ruby, I'm really not familiar with the tooling. 

Is there any goto refactoring tool for this kind of job or do you just grep the code looking for some fetch occurrences?